mitodl/micromasters
# Added staff-to-learner email with link in learner chip

#### What are the relevant tickets?
Closes #2641

#### What's this PR do?
Implements the API endpoint to send a staff-to-learner email, and adds the link on the learner chip to launch the email dialog

#### How should this be manually tested?
From the /learners page, hover over a learner's name, click the email link, send an email, and repeat for a couple of other learners. Make sure `MAILGUN_RECIPIENT_OVERRIDE` is set to your email address

#### Where should the reviewer start?
LearnerResult.js and LearnerChip.js
